s2n.mk

```diff
@@ -116,6 +116,10 @@ ifdef S2N_DEBUG
 	CFLAGS += ${DEBUG_CFLAGS}
 endif
 
+ifdef S2N_GDB
```

Comment:
Maybe call this `S2N_GDB_NO_OPTIMIZATION` to be more clear?

Reply:
I would argue the shorter name is better. I don't think the specificity is necessary in this case since we might add more compiler options in the future.